Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSPatchSensitivity.py

```diff
@@ -109,8 +109,8 @@ def __patch_workspace(self, tube_in_input_ws, in_ws, patch_ws):
                 id_to_fit.append(detector_id)
             elif not inputDetInfo.isMasked(detector_idx):
                 id_to_calculate_fit.append(detector_id)
```

Comment:
@ricleal Sorry for the confusion. I hand't looked at the surrounding code but I wasn't sure if `id_to_calculate_fit.append(detector_id)` on line 111 should be `id_to_calculate_fit.append(detector_idx)` as it was the `idx` that was not masked.